tests/installation/grub_utils.pm

```diff
@@ -0,0 +1,82 @@
+package grub_utils;
```

Comment:
Seems to me as if `grub_utils` is not a test module, so shouldn't `grub_utils.pm` be located in `lib/` instead of in `tests/installation/`?

Reply:
Yep. Good note. I'll move the module.

Reply:
Added the doc and moved `grub_test` to lib. VR passed: https://openqa.suse.de/tests/5834135